### PR TITLE
fix(DB/Creature): Spawn and Script Pool of Souls

### PR DESCRIPTION
**Pull request by Kitzunu**
## Changes Proposed:
-  Originally from TC, can however not find the original author
-  

## Issues Addressed:

- Closes 

## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1675980483756273400.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+(125921, 18766, 558, 3, 1, 0, 40.4961, -134.312, -139.589, 5.74213, 7200, 0, 0, 1, 0, 0, 0);
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`IN (18766);
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN(18766) AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
+(18766, 0, 0, 0, 10, 0, 100, 0, 0, 60, 1000, 1000, 0, 11, 32889, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Pool of Souls - Ooc los - Cast Grasp of Death');
+
+DELETE FROM `creature_template_addon` WHERE `entry`= 20316;
+INSERT INTO `creature_template_addon` (`entry`, `auras`) VALUES
+(20316, "35841 35850"); -- Raging Soul
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1675980483756273400.sql:17]: double quotes

**Comment by Gultask** [comment]
image

From Build 10.0.0.46366:

(@CGUID+78, 18766, 558, 3790, 3790, 3, 1, 0, 0, 113.589447021484375, -132.944290161132812, -139.588531494140625, 2.722713708877563476, 7200, 0, 0, 1365, 852, 0, 0, 0, 0, 46366), -- 18766 (Area: 3790 - Difficulty: 1)
(@CGUID+87, 18766, 558, 3790, 3790, 3, 1, 0, 0, 40.78951263427734375, -191.962173461914062, -139.588485717773437, 3.892084121704101562, 7200, 0, 0, 1365, 852, 0, 0, 0, 0, 46366), -- 18766 (Area: 3790 - Difficulty: 1)
(@CGUID+95, 18766, 558, 3790, 3790, 3, 1, 0, 0, 40.49610519409179687, -134.312164306640625, -139.5885009765625, 5.742133140563964843, 7200, 0, 0, 1365, 852, 0, 0, 0, 0, 46366), -- 18766 (Area: 3790 - Difficulty: 1)
(@CGUID+101, 18766, 558, 3790, 3790, 3, 1, 0, 0, 113.8101730346679687, -191.400970458984375, -139.588577270507812, 2.914699792861938476, 7200, 0, 0, 1365, 852, 0, 0, 0, 0, 46366), -- 18766 (Area: 3790 - Difficulty: 1)

MovementFlags: 1536 (DisableGravity, Root)
Flags: 33554432

They can be targeted by players

File: data/sql/updates/pending_db_world/rev_1675980483756273400.sql (hunk 1 above)
Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1675980483756273400.sql:17]: Yea it is a string, which means it doesnt matter

Comment (Nyeriah) [inline data/sql/updates/pending_db_world/rev_1675980483756273400.sql:17]:
```suggestion
(20316, '35841 35850'); -- Raging Soul
```

**Review by Gultask — APPROVED** [review]

**Pull request merged**